notebooks/inference_SavedModel_VertexEndpoint.ipynb

```diff
@@ -0,0 +1,786 @@
+{
```

Comment:
<p>Not bad. </p><p> </p><p>Maybe resize the plot?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deep-diver/semantic-segmentation-ml-pipeline/pull/48/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>